resources/styles/components/reference-book/page-navigation-controls.less

```diff
@@ -0,0 +1,84 @@
+.reference-book {
+  .page-wrapper > .nav {
+    position: fixed;
+    height: 100%;
+    width: calc(~"50% - " @reference-book-page-width / 2 );
+    top: 50px;
+
+    display: -webkit-flex;
```

Comment:
Lesshat has some neat stuff, but in my opinion the `calc`  mixin doesn't read any better than just using `calc` directly:  https://github.com/madebysource/lesshat/blob/master/README.md#calc

And since `calc` is supported by all of our target browsers (http://caniuse.com/#search=calc) I don't see the reason for the vendor prefixes.

I was hoping lesshat would have support for css triangles but looks like it doesn't